src/components/tasks/card/card.module.scss

```diff
@@ -142,6 +142,18 @@ $seeMoreTasksShadow: #595959cc;
     border-radius: 1rem;
 }
 
+.progressBarContainer {
+    display: flex;
+    flex-direction: column;
+}
+
+.changeProgressText {
+    cursor: pointer;
+    color: $taskTagLevelBorder;
+    font-weight: 700;
+    padding-bottom: 0.6rem;
```

Comment:
Nit: This is @yesyash said me once while he review my pr...below is yashraj wording I just copy and paste
I want to share this learning to you also

Using random values like these makes the design feel inconsistent. Even if we do not have a design system in place please use multiples of 4 (4, 8, 16, 20, 24...) converted into rem units (0.25rem, 0.5rem, 0.75rem ...) for font sizes and spacing units (padding / margin) as it will make the layout look neat and consistent.

-> https://github.com/Real-Dev-Squad/website-www/pull/456#discussion_r1165893502

Reply:
so 1 unit = 4px according to design guidelines
suppose the padding-top value needs to be 24px according to the Figma design
If  we take out how many units will be 24px so it will be 6 units
and 1rem = 16px so 24px / 16px = 1.5rem
so can I use padding-top as 1.5rem
but we usually make it even like 1.6rem
if it's 0.25 then we can make it 0.24 or 0.26 to give it even number feel

Reply:
```
but we usually make it even like 1.6rem
``` 

Who are we here and what is this even number feel? 
Any reason for rounding off the number?

Reply:
Yes, there is a reason for rounding off the number because I was told by Vaibhav in my last feature that instead of 1.5rem we should take 1.4rem or 1.6rem to maintain symmetry because it's difficult with odd numbers to bring things in symmetry.

Reply:
@yesyash please reply on this??

Reply:
@yesyash can you reply or should we resolve this???

Reply:
@Pratiyushkumar If you're having a 4px or 8px scale (I think 8px in RDS) it's always important you abide by the values of the system when building your designs. 

We also generally use the 4px or 8px scale because most of the screens are divisible by them and they help us have a consistent scaling factor across screens. I would highly recommend you see some major brand design guidelines (Ex: Apple, Airbnb) where you see this pattern a lot.  

And in the new design system that is being built by Abhijeet, you will be able to see the same pattern.

Reply:
fixed